index.html

```diff
@@ -10,7 +10,7 @@
     <meta name="description" content="Polaroid a CSS component library for your awesome UI." />
 
     <!-- links -->
-    <link rel="stylesheet" href="/documentation/documentation.css" />
+    <link rel="stylesheet" href="/pages/documentation.css" />
     <title>⚒ Polaroid | Home</title>

```

Comment:
Since it is a stylesheet... It does not make sense to me as we add this in file in pages, it can be in styles